Packs/Humio/Integrations/Humio/Humio.yml

```diff
@@ -0,0 +1,356 @@
+commonfields:
+  id: Humio
+  version: -1
+name: Humio
+display: Humio
+category: Analytics & SIEM
+description: Integration with Humio
+configuration:
+- display: Humio URL
+  name: url
+  defaultvalue: https://cloud.humio.com
+  type: 0
+  required: true
+- display: User API token
+  name: API-key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Query to use to fetch incidents
+  name: queryParameter
+  defaultvalue: ""
+  type: 0
+  required: false
+  additionalinfo: Write the query to use to fetch incidents (only used for fetch-incidents)
+- display: Fetch incidents from repository
+  name: queryRepository
+  defaultvalue: ""
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: Fetch incidents from
+  name: queryStartTime
+  defaultvalue: 24h
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: TimeZoneOffset in Minutes
+  name: queryTimeZoneOffsetMinutes
+  defaultvalue: "0"
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: humio-query
+    arguments:
+    - name: repository
+      required: true
+      description: Repository to search
+    - name: queryString
+      required: true
+      description: Query string to use
+    - name: start
+      description: Relative or absolute (epoch)
+      defaultValue: 24h
+    - name: end
+      description: Relative or absolute (epoch)
+      defaultValue: now
+    - name: isLive
+      description: Answer with true, 1, t, y or yes
+      defaultValue: "false"
+    - name: timeZoneOffsetMinutes
+      description: TimeZoneOffset in Minutes (default 0)
+      defaultValue: "0"
+    - name: arguments
+      description: Additional arguments
+    outputs:
+    - contextPath: Humio.Query
+      description: Query output
```

Comment:
Define the output fields of Query, especially returned `id` because it should be used later in poll command

Reply:
The query command does not have an id, as it queries directly without creating a job, so this is generic and could look like anything.